.github/workflows/backwards-compatibility-commenter.yml

```diff
@@ -0,0 +1,20 @@
+name: Comments for ensuring backward compatibility
+
+on:
+  pull_request_target:
+    paths:
+      - 'ace.d.ts'
+      - 'ace-modes.d.ts'
+      - 'ace-extensions.d.ts'
+
+jobs:
+  comment:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Comment on PR
+        uses: peter-evans/create-or-update-comment@v1
+        with:
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            One of the public type files has been updated, plase make sure there are no backwards incompatible changes done in the PR.
```

Comment:
Do you think we should mention what we consider a breaking change (modifications and deletions) or do you think that's obvious?